Readme.md

```diff
@@ -79,6 +79,15 @@ https://nodejs.org/en/download/
 
 `$ npm install`
 
+В директории `src/frontend` возможно выполнить следующие скрипты:
```

Comment:
Also the PR contains next updates:
- https://htmlacademy-vue.slack.com/archives/C025TFESWQY/p1625903636016900
- https://htmlacademy-vue.slack.com/archives/C025TFESWQY/p1625479808013700